parcels/particleset.py

```diff
@@ -223,25 +223,34 @@ def remove(self, indices):
         return particles
 
     def execute(self, pyfunc=AdvectionRK4, endtime=None, runtime=None, dt=1.,
-                interval=None, recovery=None, output_file=None,
-                show_movie=False):
+                outputdt=None, moviedt=None, recovery=None, output_file=None, movie_background_field=None):
         """Execute a given kernel function over the particle set for
         multiple timesteps. Optionally also provide sub-timestepping
         for particle output.
 
         :param pyfunc: Kernel function to execute. This can be the name of a
                        defined Python function or a :class:`parcels.kernel.Kernel` object.
                        Kernels can be concatenated using the + operator
-        :param endtime: End time for the timestepping loop
+        :param endtime: End time for the timestepping loop.
+                        It is either a datetime object or a positive double.
         :param runtime: Length of the timestepping loop. Use instead of endtime.
-        :param dt: Timestep interval to be passed to the kernel
-        :param interval: Interval for inner sub-timestepping (leap), which dictates
-                         the update frequency of file output and animation.
+                        It is either a timedelta object or a positive double.
+        :param dt: Timestep interval to be passed to the kernel.
+                   It is either a timedelta object or a double.
+                   Negative value means backward in time simulation.
+        :param outputdt: Interval for inner sub-timestepping (leap), which dictates
+                         the update frequency of file output.
+                         It is either a timedelta object or a positive double.
+                         None value means no file output.
```

Comment:
I'm thinking; why don't we take the `outputdt` from the output_file? Would that not make more sense? Because now users have to provide *both* a `outputdt` and an `output_file` if they want output. 

If instead they specify `outputdt` on `ParticleFile.__init__`, then here it requires only a `output_file`. That's also how we do `repeatdt`, by the way